### ISSUE TITLE
No chapter names

### ISSUE THREAD

**Issue by Zehirmahn** [yt-dlp/yt-dlp#7636]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

--embed-chapters works to add chapter timestamps into the video file, but it doesn't include the actual chapter names.

I think I have identified the problem : 
- DO NOT use "-add-metadata 1" in the FFMPEG command to merge the .meta file to the video file.
- Use this instead : 

-> ffmpeg -i "in.ext" -i "in.meta" -c copy -map 0 "out.ext"

You can access the .meta file to test yourself by using the --embed-chapters option and renaming/removing the ffmpeg.exe, so yt-dlp doesn't know what to do with this .meta file and let it be.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [x] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--embed-chapters', 'https://www.youtube.com/watch?v=wAckkB-dY2M']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2023-07-16-git-c541ecf0dc-essentials_build-www.gyan.dev (setts)
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=wAckkB-dY2M
[youtube] wAckkB-dY2M: Downloading webpage
[youtube] wAckkB-dY2M: Downloading ios player API JSON
[youtube] wAckkB-dY2M: Downloading android player API JSON
[youtube] wAckkB-dY2M: Downloading m3u8 information
[youtube] wAckkB-dY2M: Downloading MPD manifest
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] wAckkB-dY2M: Downloading 1 format(s): 248+251
[debug] Invoking dashsegments downloader on "https://manifest.googlevideo.com/api/manifest/dash/expire/1689737782/ei/1gW3ZNfQC4qsp-oPh6qc8A8/ip/2a02%3A842a%3A3b2%3Abe01%3Ab078%3A27de%3Aced%3Acded/id/c00724901f9d6363/source/youtube/requiressl/yes/playback_host/rr1---sn-n4g-ator.googlevideo.com/mh/dE/mm/31%2C29/mn/sn-n4g-ator%2Csn-hgn7rn7r/ms/au%2Crdu/mv/m/mvi/1/pl/47/hfr/all/as/fmp4_audio_clear%2Cwebm_audio_clear%2Cwebm2_audio_clear%2Cfmp4_sd_hd_clear%2Cwebm2_sd_hd_clear/initcwndbps/1221250/spc/Ul2Sq3QgWvAtkaORJwB0B7oUqwo5egAzamu8sRz_dA/vprv/1/mt/1689715755/fvip/1/keepalive/yes/fexp/24007246%2C24362688%2C51000024/beids/24350018/itag/0/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Chfr%2Cas%2Cspc%2Cvprv%2Citag/sig/AOq0QJ8wRQIgSLmjTMI0TCFOIr_rsddS6vT2D0buCB8JD2JO860SW1cCIQDlevUAo2mvODOlSHyDaq28Y_828LBf1VAtfZY4mhfsWw%3D%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/AG3C_xAwRQIgLlHxcleJiPBzOvaf1O6UH1YiougxgdmErEBYaZ20CI4CIQCcF_-FPr0BypuKwk-QEr2KPLO1mVKObuULI1vB8GnGvQ%3D%3D"
[dashsegments] Total fragments: 25
[download] Destination: Chapters test. [wAckkB-dY2M].f248.webm
[debug] File locking is not supported. Proceeding without locking
[download] 100% of   10.46MiB in 00:00:04 at 2.12MiB/s
[debug] Invoking http downloader on "https://rr1---sn-n4g-ator.googlevideo.com/videoplayback?expire=1689737782&ei=1gW3ZIy9Muedp-oPirGh-AU&ip=2a02%3A842a%3A3b2%3Abe01%3Ab078%3A27de%3Aced%3Acded&id=o-AAtj1KVOmIuUKm17iRcBY20WM1daFYiJjaP_Kpfhr8vs&itag=251&source=youtube&requiressl=yes&mh=dE&mm=31%2C29&mn=sn-n4g-ator%2Csn-hgn7rn7r&ms=au%2Crdu&mv=m&mvi=1&pcm2cms=yes&pl=47&initcwndbps=1221250&spc=Ul2SqzFVepOhZA4Q-0h89icdiblXLYo&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=1480922&dur=127.261&lmt=1605991946037719&mt=1689715755&fvip=1&keepalive=yes&fexp=24007246&beids=24350018&c=ANDROID&txp=6211222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIgbvE76peebATnO3Bf6M-A0lQsuR8oFQZRMcQgj5eMpWICIQDfeEDD1IztSKWbTX6a6XyXlczgxNjzPWnysw-J8bwyfA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRgIhAP__SXVpXYI-afZrdnAwv5Ya9CCfyS9BS_7hx-fiPowEAiEAvxGmFof091XaRs0G5TlNNecfkyfuNAvxvL90fEurDX8%3D"
[download] Destination: Chapters test. [wAckkB-dY2M].f251.webm
[download] 100% of    1.41MiB in 00:00:00 at 14.62MiB/s
[Merger] Merging formats into "Chapters test. [wAckkB-dY2M].webm"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Chapters test. [wAckkB-dY2M].f248.webm" -i "file:Chapters test. [wAckkB-dY2M].f251.webm" -c copy -map "0:v:0" -map "1:a:0" -movflags "+faststart" "file:Chapters test. [wAckkB-dY2M].temp.webm"
Deleting original file Chapters test. [wAckkB-dY2M].f251.webm (pass -k to keep)
Deleting original file Chapters test. [wAckkB-dY2M].f248.webm (pass -k to keep)
[Metadata] Adding metadata to "Chapters test. [wAckkB-dY2M].webm"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Chapters test. [wAckkB-dY2M].webm" -i "file:Chapters test. [wAckkB-dY2M].meta" -map 0 -dn -ignore_unknown -c copy -map_metadata 1 -movflags "+faststart" "file:Chapters test. [wAckkB-dY2M].temp.webm"
```

**Comment by bashonly** [comment]
Duplicate of #6859

It's a regression in ffmpeg. Downgrade to ffmpeg 6.0 or earlier

**Comment by Zehirmahn** [comment]
Sorry for the duplicate, i tried to search...
I'm not sure ffmpeg will adress this issue soon, and if it can work with another (simplier) cmd why not implement it in yt-dlp?

**Comment by bashonly** [comment]
yt-dlp depends on ffmpeg for all metadata/chapter embedding. The simplest "fix" is to downgrade to ffmpeg 6.0
